controllers/forum.js

```diff
@@ -0,0 +1,77 @@
+const   express     = require('express'),
+        Forum       = require('../models/forum'),
+        router      = express.Router();
+
+        //FIND ALL
+        router.get('/', async (request,response)=>{            
+            await Forum.find({},(error,allForums)=>{
+                if(error)
+                {
+                    console.log(error);
+                    response.redirect('/');
+                }
+                else
+                {
+                    response.send(allForums);
+                }
+            });
+        });
+
+        //CREATE 
+        router.post('/', async (request,response)=>{            
```

Comment:
Replace all "request" and "response" with "req", "res"

Reply:
I used them initially but I thought we should be using meaningful variables that's why I kept it that way

Reply:
I used them initially but I thought we should be using meaningful variables that's why I kept it that way